src/lib/components/DeckGLMap/layers/wells/wellsLayer.ts

```diff
@@ -1,9 +1,23 @@
 import { CompositeLayer } from "@deck.gl/core";
+import { CompositeLayerProps } from "@deck.gl/core/lib/composite-layer";
 import { GeoJsonLayer } from "@deck.gl/layers";
 
-export default class WellsLayer extends CompositeLayer {
+export interface WellsLayerProps<D> extends CompositeLayerProps<D> {
+    pointRadiusScale: number;
+    lineWidthScale: number;
+    outline: boolean;
+}
+
+function getOutlineColor() {
+    return [0, 0, 0, 255];
+}
+
+export default class WellsLayer extends CompositeLayer<
+    unknown,
+    WellsLayerProps<unknown>
+> {
     renderLayers() {
-        let properties = {
+        const properties: any = {
```

Comment:
I'm open to suggestions how to solve that :D

Reply:
I've checked locally, and it looks like the problem is with getFillColor, getting the color out of `d`, since it's `unknown`.
It seems to work if you define a data structure for the input well data and use it instead of `unknown`. I've quickly added something based on the fields in volve_wells.json, you might need to add more stuff:
 ```
export interface WellDataType {
    type: string;
    geometry: Record<string, unknown>;
    properties: {
        name: string;
        color: RGBAColor;
    };
}
```